modules/assim.sequential/R/sda.enkf_MultiSite.R

```diff
@@ -472,8 +501,12 @@ sda.enkf.multisite <- function(settings, obs.mean, obs.cov, Q = NULL, restart=F,
     save(site.locs, t, FORECAST, ANALYSIS, enkf.params, new.state, new.params,
          out.configs, ensemble.samples, inputs, Viz.output,
          file = file.path(settings$outdir,"SDA", "sda.output.Rdata"))
-    #writing down the image - either you asked for it or nor :)
```

Comment:
Now there is no plotting function called in the main SDA code. Use `control$TimeseriesPlot` inside the `if` so the user can control the plotting.

Reply:
cool! thanks

Reply:
fixed